content/lang/de/guide/api-reference.md

```diff
@@ -0,0 +1,129 @@
+---
+name: API-Referenzierung
+permalink: '/guide/api-reference'
+---
+
+# API-Referenzierung
+
+## `Preact.Component`
+
+`Component` ist eine Basisklasse, die normalerweise als Subklasse zum Erstellen von kraftvollen Preact-Komponenten verwendet wird.
+
+### `Component.render(props, state)`
+
+Die `render()`-Funktion ist voraussetzend für alle Komponenten. Sie kann die props und states der Komponente inspizieren und sollte ein Preact-Element oder `null` zurückgeben.
+
+```jsx
+import { Component } from 'preact';
+
+class MeineKomponente extends Component {
+	render(props, state) {
+		// props === this.props
+		// state === this.state
+
+		return <h1>Hello, {props.name}!</h1>;
+	}
+}
+```
+
+### Lebenszyklusmethoden
+
+> _**Tipp:** Wenn du eigene HTML5-Elemente verwendert hast: diese ähneln sich mit den `attachedCallback`- und `detachedCallback`-Lebenszyklusmethoden._
+
+Preact ruft die nachfolgenden Lebenszyklusmethoden auf, falls sie für eine Komponente definiert sind:
+
+| Lebenszyklusmethoden        | Wann sie aufgerufen wird                             				 |
+|-----------------------------|--------------------------------------------------------------|
+| `componentWillMount`        | bevor die Komponente an das DOM eingehanden wird					   |
```

Comment:
bevor die Komponente an das DOM eingehanden wird
`eingehanden`?

Wäre das technisch richtig?:
bevor die Komponente in das DOM eingebunden wird